quickstart.md

```diff
@@ -18,7 +18,41 @@ Quick Start Guide
 
 This is everything you need to run DL4J examples and begin your own projects.
 
-We recommend that you join our [Gitter Live Chat](https://gitter.im/deeplearning4j/deeplearning4j). Gitter is where you can request help and give feedback, but please do use this guide before asking questions we've answered below. If you are new to deep learning, we've included [a road map for beginners](./deeplearningforbeginners.html) with links to courses, readings and other resources. 
+We recommend that you join our [Gitter Live Chat](https://gitter.im/deeplearning4j/deeplearning4j). Gitter is where you can request help and give feedback, but please do use this guide before asking questions we've answered below. If you are new to deep learning, we've included [a road map for beginners](./deeplearningforbeginners.html) with links to courses, readings and other resources. If you need an end to end tutorial to get started (including setup), then please go to our [getting started](http://deeplearning4j.org/gettingstarted)
+
+Overview
+===================================
+
+Deeplearning4j is a "pick and choose with everything available" framework. That means *everything* is made available for you from the beginning. Deeplearning4j has several sub projects for you to build end to end applications.
+Deeplearning4j is not tensorflow or pytorch. See [our comparison page](https://deeplearning4j.org/compare-dl4j-torch7-pylearn)
+
+
+If you're coming here looking to just deploy models to production, you might like our [model import from keras](http://deeplearning4j.org/keras)
+
+Deeplearning4j has several submodules. This ranges from a visualization UI to distributed training on spark. For an overview of these modules, please look at our [examples](https://github.com/deeplearning4j/dl4j-examples)
+
+To get started on a simple desktop app, you need 2 things. An [nd4j backend](http://nd4j.org/backend.html) and deeplearning4j-core. For more code, see the [simpler examples submodule](https://github.com/deeplearning4j/dl4j-examples/blob/master/dl4j-examples/pom.xml#L64)
+
+If you want a super flexible deep learning api, there are 2 ways you can go.  You can use nd4j standalone See our [nd4j examples](https://github.com/deeplearning4j/dl4j-examples/tree/master/nd4j-examples) or the [computation graph api](http://deeplearning4j.org/compgraph)
+
+If you want to do distributed training on spark, you can see our [spark page](http://deeplearning4j.org/spark]()
```

Comment:
you can see our > see our (delete 'you can')
ending period missing
Are the closing parens () needed?